tests/arduino_lib/README.md

```diff
@@ -0,0 +1,8 @@
+Expected result
```

Comment:
The README is copy-pasted from `tests/driver_at`.

Reply:
I have removed the file.